src/enhetsportefolje/enhet-kolonner.tsx

```diff
@@ -139,7 +146,8 @@ function EnhetKolonner({className, bruker, enhetId, filtervalg, valgteKolonner,
             <DatoKolonne
                 className="col col-xs-2"
                 dato={nesteUtlopsdatoEllerNull(valgteAktivitetstyper)}
-                skalVises={!!valgteAktivitetstyper && filtervalg.tiltakstyper.length === 0 && valgteKolonner.includes(Kolonne.UTLOP_AKTIVITET)}
+                // skalVises={!!valgteAktivitetstyper && filtervalg.tiltakstyper.length === 0 && valgteKolonner.includes(Kolonne.UTLOP_AKTIVITET)}
```

Comment:
ska den utkommenterade koden behållas ?

Reply:
Ja, koden er ikke egentlig ferdig, men jeg må merge den i dev for å teste den i q0 siden jeg mangler data lokalt -.-